src/tree/StaticWebAppTreeItem.ts

```diff
@@ -4,11 +4,12 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { WebSiteManagementClient, WebSiteManagementModels } from "@azure/arm-appservice";
+import { ResourceManagementModels } from '@azure/arm-resources';
```

Comment:
I'd recommend `import type` to help keep this dev-time. Could add for the import above this for `@azure/arm-appservice` too.